pkg/kubelet/server/remotecommand/httpstream.go

```diff
@@ -176,6 +189,56 @@ type protocolHandler interface {
 	waitForStreams(streams <-chan streamAndReply, expectedStreams int, expired <-chan time.Time) (*context, error)
 }
 
+// v3ProtocolHandler implements the V3 protocol version for streaming command execution.
+type v3ProtocolHandler struct{}
```

Comment:
I tried adding it to v2. If you try to create the resize stream and the server isn't expecting it, you'll end up stalling 30 seconds (default value) and then the stream creation will time out. The easiest way forward seemed to be v3.

It should be easier when we switch to http/2, since we won't be creating streams and everything will be multiplexed in the request/response body.